ramp_onboarding_ux.py

```diff
@@ -256,6 +313,30 @@ def index() -> Any:  # pylint: disable=too-many-branches
             "googleMapsApiKey": app.config["GOOGLE_MAPS_FRONTEND_API_KEY"],
             "googleClientId": app.config["GOOGLE_CLIENT_ID"],
             "googleOneTapLoginUri": url_for("verify_google_onetap", _external=True),
+            "showAdvancedOptions": not session["is_student"],
+            "departmentOptions": departments,
+            "departmentId": default_department,
+            "locationOptions": locations,
+            "locationId": default_location,
+            "roleOptions": {
+                "BUSINESS_USER": {
+                    "label": "Employee",
+                    "enabled": True,
+                },
+                "BUSINESS_BOOKKEEPER": {
+                    "label": "Bookkeeper",
+                    "enabled": True,
+                },
+                "BUSINESS_ADMIN": {
+                    "label": "Admin",
+                    "enabled": False,  # TODO waiting on ramp developer support to clarify if this is supposed to work  # noqa  # pylint: disable=fixme
```

Comment:
Ticket opened here: https://support.ramp.com/hc/en-us/requests/679791

Depending on when I get a review vs a response on the ticket I'll probably go ahead and merge this and then update it later.